Automattic/wp-calypso
# Update Calypso Colors

This branch updates all Calypso components and all color schemes with the [recently refreshed](https://color-studio.blog) palette colors.

## Notable changes

### Color schemes

Color scheme properties have been refactored:

* Color scheme stylesheets have been split into separate partials for better readability.
* All themable color properties are now prefixed with `--color-` while generic colors are directly outsourced to Color Studio properties (which are prefixed with `--studio-` and can’t be overwritten).
* Most color references in the code base have been outsourced to theme properties.
* Nightfall’s accent color has been changed from Purple to Blue due to WooCommerce folks’ wish to limit WordPress.com’s use of purple.
* All color combinations should be 100% accessible now.

### Components

Some components had to receive dedicated but non-breaking color treatments or refinements for better overall consistency:

* Badge
* Banner
* Color Scheme Picker
* Credit Card
* Jetpack Logo
* Push Notification Illustration
* Masterbar
* Notice
* Sidebar

### Other

* All the remaining color variables defined in SCSS have been phased out completely in favor of CSS custom properties.
* Updated SVG files for all plan icons and color scheme thumbnails. _No other SVG files have been touched._
* Removed unused code bits (where identified).

## Testing instructions

* Ensure color scheme switching works as intended.
* Ensure the components listed above are displayed correctly with all color schemes and under all supported browsers.
* Ensure this update doesn’t introduce any visual regressions.
